modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -41,6 +41,42 @@ apt install apt-transport-https
 
 == Add Software Channels
 
+Before you can register {debian} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+ifeval::[{suma-content} == true]
+
+The products you need for this procedure are:
+
+[[debian-channels-wizard]]
+[cols="1,1", options="header"]
+.Debian Products - WebUI
+|===
+| OS Version  | Product Name
+| {debian} 9  | SUSE Linux Enterprise Client Tools Debian 9 amd64
+| {debian} 10 | SUSE Linux Enterprise Client Tools Debian 10 amd64
```

Comment:
The Client Tools are just a child product. Please check how you described Ubuntu 20.04. Both Debian versions are setup the same way.